### PR TITLE
Translate status page.

### PR DESCRIPTION
**Pull request by chijoy**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -430,19 +437,20 @@ en:
...
-        # num_in_household: "Number in household"
-        # num_of_dependants: "Number of dependants"
-    # errors:
-    #   models:
-    #     client:
-    #       attributes:
-    #         ssn:
-    #         dob:
-    #         num_of_dependants:
-    #         num_in_household:
```
Comment (heydanhey) [inline config/locales/en.yml]: Hmm, interesting. I understand we you're removing this, it's not being used, right? But if we wanted to add the internationalization to the active record error messages, is this How'd we do it?  Did we ever try it?

File: app/views/devise/shared/_links.html.erb [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+  <%= link_to I18n.t("devise.sessions.client.sign_up"), new_registration_path(resource_name) %><br />
 <% end -%>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
+  <%= link_to I18n.t("devise.sessions.client.forgot_your_password"), new_password_path(resource_name) %><br />
 <% end -%>
```
Comment (heydanhey) [inline app/views/devise/shared/_links.html.erb]: Nice find!

File: app/views/clients/status.html.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <div class="client-status">  
   <h1>
-    Application Status for: <strong class="text-primary"><%= @client.full_name %></strong><br />
+    <%= I18n.t "client_status.application_status_for" %><strong class="text-primary"><%= @client.full_name %></strong><br />
     <small>
-      Thank you for your interest in LUCHA's services. See the steps below to complete your application.
+      <%= I18n.t "client_status.thank_you" %>
     </small>
```
Comment (heydanhey) [inline app/views/clients/status.html.erb]: This is awesome! I knew the status page was the biggest one left to add this stuff to, way to go!

File: config/locales/en.yml (hunk 1 above)
Comment (chijoy) [inline config/locales/en.yml]: I tried it, and i could never figure it out. You're right, I should have just kept it commented out...

File: app/views/clients/status.html.erb (hunk 3 above)
Comment (chijoy) [inline app/views/clients/status.html.erb]: I think we just need to add translations for the employee section that you added.

Comment (heydanhey) [inline app/views/clients/status.html.erb]: You're right, but that page is for employees and I'm not sure it's in our scope to translate the pages that are not client facing? What do you think is the easiest way to have LUCHA fill in the translations? I was thinking there is probably some tool already built for this very thing.  Here's a few things I found: [i18n-tasks](https://github.com/glebm/i18n-tasks), [Locale](https://www.localeapp.com) - we should look into them or maybe even spin up a quick app we could share?

**Pull request merged**